### PR TITLE
Change the height of network dropdown

### LINKED ISSUE #192: Use the same size drop down inputs in the navigation bar
At the moment the two drop-downs inputs have a slightly different height

Screenshot 2023-06-06 at 10 43 09 AM

### PR DESCRIPTION
**Pull request by asnaith**
closes #192 

**Before**

Screenshot 2023-06-06 at 4 20 22 PM

**After**

Screenshot 2023-06-06 at 4 20 55 PM

### CODE REVIEW

**Review by Tbaut — APPROVED** [review]
Oops, found an issue on FF
image

**Review by Tbaut — CHANGES_REQUESTED** [review]
oops found an issue
image

File: packages/ui/src/components/NetworkSelection.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default styled(NetworkSelection)(
     
     .MuiSelect-select {
       display: flex;
-      height: 1rem;
+      max-height: 1rem;
```
[the remark this answers is not in the document]
Comment (asnaith) [reply packages/ui/src/components/NetworkSelection.tsx]: @Tbaut Yep, looks good with this change. Correct height and the logo looks good in Firefox, brave, chrome and Safari.

**Review by Tbaut — APPROVED** [review]
:+1:

**Pull request merged**